lib/middlewares/responseGenerator.js

```diff
@@ -142,6 +156,10 @@ function processAsynchronousRequest(req, res, next) {
             case 'GPS':
                 moduleAnswer = singleValueResponse(module);
                 break;
+            case 'L1':
```

Comment:
Same answer is expected. This part of the code belongs to the next PR I'm going to create, as it is part of the actuators for the Hackathon, not part of the integration of the Black Button (I worked in parallel with both and mixed them a bit).

NTC.